src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/ReportsServiceTest.java

```diff
@@ -49,13 +50,30 @@ public void should_map_repo_result_properly() {
             );
     }
 
+    @Test
+    public void should_filter_out_test_jurisdiction_when_requested() {
```

Comment:
To make it perfect, we could also check if `resultWithoutTestJurisdiction` filters out BULKSCAN and not the other jurisdiction.

Reply:
added extra assertion